pds_doi_core/input/input_util.py

```diff
@@ -67,6 +67,9 @@ def _parse_rows_to_doi_meta(self, xl_sheet):
             doi_record['product_type'] = 'Collection'
             doi_record['product_type_specific'] = row['product_type_specific']
             doi_record['related_identifier'] = row['related_resource']
```

Comment:
Sorry about that again, we dont need to handle the submitter_email here. It is going to be in the arguments of the command line.

Reply:
The submitter_email is now part of the command line argument.